src/components/Layout/RightPaneLayout/useRightPane.tsx

```diff
@@ -0,0 +1,27 @@
+import { ReactNode, useCallback } from "react";
+import { useRightPaneContext } from "./RightPaneContext";
+
+interface OpenRightPaneOpts {
+  content: ReactNode;
+}
+
+export interface UseRightPaneHook {
+  /** Opens a right pane */
+  // openRightPane: (opts: OpenRightPaneOpts) => void;
+  openRightPane: (opts: OpenRightPaneOpts) => void;
+  /** Closes the right pane */
+  closeRightPane: () => void;
+}
+
+export function useRightPane(): UseRightPaneHook {
+  const { openInPane, closePane } = useRightPaneContext();
+  const openRightPane = useCallback((opts: OpenRightPaneOpts) => {
+    openInPane(opts);
+  }, []);
+  const closeRightPane = useCallback(() => closePane(), []);
+
+  return {
+    openRightPane,
+    closeRightPane,
+  };
```

Comment:
**super nit:** This is a good opportunity to go from 4 lines of code to 1. 
```suggestion
  return { openRightPane, closeRightPane };
```

Reply:
hm - odd perhaps auto formatting..